src/state/store.ts

```diff
@@ -4,5 +4,6 @@ import { reducer } from './reducer'
 
 export const store = configureStore({
   reducer,
-  devTools: true
+  devTools: true,
+  middleware: getDefaultMiddleware => getDefaultMiddleware({ serializableCheck: false })
```

Comment:
what's going on here? this sounds like we're putting things that can't be serialized in the store?

Reply:
This check only runs in dev. Its removed in prod (hence why mobile has no issues). It should be removed to keep people from putting state in the store that cant be serialized. I was testing to see if it would speed up the app but no luck. Will remove.